etna/search/views.py

```diff
@@ -526,45 +526,20 @@ def get_selected_filters(self, form: Form) -> Dict[str, List[Tuple[str, str]]]:
         if filter_keyword := form.cleaned_data.get("filter_keyword"):
             return_value.update({"filter_keyword": [(filter_keyword, filter_keyword)]})
 
-        # get form error messages
-        if error_dict := json.loads(form.errors.as_json()):
-            for dict_values in error_dict.values():
-                for item in dict_values:
-                    form_error_messages.append(item["message"])
-
-        if opening_start_date := form.cleaned_data.get("opening_start_date"):
-            # if both dates have valid values but invalid when together
-            if (
-                CUSTOM_ERROR_MESSAGES.get("invalid_date_range")
-                not in form_error_messages
-            ):
-                return_value.update(
-                    {
-                        "opening_start_date": [
-                            (
-                                opening_start_date,
-                                opening_start_date.strftime(
-                                    "Record Opening From: %d-%m-%Y"
-                                ),
-                            )
-                        ]
-                    }
-                )
-
-        if opening_end_date := form.cleaned_data.get("opening_end_date"):
-            # if both dates have valid values but invalid when together
-            if (
-                CUSTOM_ERROR_MESSAGES.get("invalid_date_range")
-                not in form_error_messages
-            ):
+        # prepare filter labels for date fields
+        for date_item in [
+            ("opening_start_date", "Record Opening from: %d-%m-%Y"),
+            ("opening_end_date", "Record Opening to:  %d-%m-%Y"),
+            ("created_start_date", "Date from: %d-%m-%Y"),
+            ("created_end_date", "Date to: %d-%m-%Y"),
+        ]:
```

Comment:
This list should be a tuple, since it doesn't need to be mutable. We should unpack the item tuples here to make the following code more readable too, e.g.:
```suggestion
        for field_name, label in [
            ("opening_start_date", "Record Opening from: %d-%m-%Y"),
            ("opening_end_date", "Record Opening to:  %d-%m-%Y"),
            ("created_start_date", "Date from: %d-%m-%Y"),
            ("created_end_date", "Date to: %d-%m-%Y"),
        ]:
```